frontend/packages/component-library/tsup.config.ts

```diff
@@ -2,11 +2,25 @@ import {defineConfig} from 'tsup';
 import {postcssModules, sassPlugin} from 'esbuild-sass-plugin';
 import {glob} from 'glob';
 import {resolve} from 'node:path';
+import {cp} from 'node:fs/promises';
 
 const entryPoints = glob.sync('./src/**/index.ts', {
   posix: true,
 });
 
+// Helper to copy styles folder after build
+async function copyStyles() {
+  const sourceDir = './src/common/styles';
+  const destDir = './dist/common/styles';
```

Comment:
Preferably we would just have a `styles` directory, so `dist/styles` rather than `dist/common/styles`

Reply:
I don't agree on this. It's a common styles and they should be available via same path they are available right now.
Changing import path will be a breaking change, we will need to go through the imports and explicitly change paths. On top of that, we'll loose semantics from the path that help consumer understand what's these styles are responsible for. I don't see why we might want to do that.